documentation/jetty-documentation/src/main/asciidoc/programming-guide/maven/jetty-maven-plugin.adoc

```diff
@@ -77,8 +77,8 @@ mvn jetty:run
 This starts Jetty and serves up your project on http://localhost:8080/.
 
 Jetty will continue to run until you stop it.
-By default, it will not automatically restart your webapp: you can force a redeploy by hitting the `Enter` key.
-Set a non-zero &lt;scan&gt; value to have jetty scan your webapp for changes and automatically redeploy.
+By default, it will not automatically restart your webapp.
+Set a non-zero `&lt;scan&gt;` value to have jetty scan your webapp for changes and automatically redeploy, or set `&lt;scan&gt;` to `0` to cause manual redeployment by hitting the `Enter` key.
```

Comment:
Replace HTML entities with correspondent characters.

Reply:
Use `kbd:[Enter]` to indicate the "Enter" key -- it will render more nicely, see https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#keyboard-button-and-menu-macros